RealmSwift-swift2.2/Tests/TestUtils.mm

```diff
@@ -53,3 +53,25 @@ void RLMAssertThrows(XCTestCase *self, dispatch_block_t block, NSString *name, N
 bool RLMHasCachedRealmForPath(NSString *path) {
     return RLMGetAnyCachedRealmForPath(path.UTF8String);
 }
+
+void RLMAssertThrowsWithReasonMatchingSwift(XCTestCase *self, dispatch_block_t block, NSString *regexString, NSString *message, NSString *fileName, NSUInteger lineNumber) {
```

Comment:
Pulling the helper functions for both frameworks into a file shared between them sounds like a good idea.